roles/scm/handlers/main.yml

```diff
@@ -8,8 +8,9 @@
 
 - name: wait cloudera-scm-server
   wait_for:
-    host={{ hostvars[scm_hostname]['inventory_hostname'] }}
+    host={{ hostvars[scm_hostname][hostname_var] }}
```

Comment:
actually here even better to just use scm_hostname:
    host={{ scm_hostname }}